packages/atomic/src/index.html

```diff
@@ -124,55 +125,13 @@
           </div>
 
           <atomic-no-results></atomic-no-results>
-          <atomic-result-list fields-to-include="liboardinteractionstyle,ytthumbnailurl">
-            <!-- Custom result templates examples -->
-            <atomic-result-template must-match-filetype="youtubevideo" must-not-match-liboardinteractionstyle="contest">
-              <template>
-                <!-- Styling result template internaly -->
-                <style>
-                  h3 {
-                    font-weight: bold;
-                    color: var(--atomic-secondary);
-                  }
-
-                  atomic-result-image[field='ytthumbnailurl'] {
-                    display: block;
-                    width: 200px;
-                  }
-
-                  .youtubevideo-container {
-                    display: flex;
-                  }
-                </style>
-                <div class="youtubevideo-container">
-                  <atomic-result-link>
-                    <atomic-result-image field="ytthumbnailurl"></atomic-result-image>
-                  </atomic-result-link>
-                  <div>
-                    <atomic-result-link></atomic-result-link>
-                    <atomic-result-text field="excerpt"></atomic-result-text>
-                  </div>
-                </div>
-              </template>
-            </atomic-result-template>
-            <atomic-result-template id="singaporeTitle">
-              <template> 🇸🇬 <atomic-result-link></atomic-result-link> </template>
-              <!-- Adding Javascript conditions -->
-              <script>
-                document.querySelector('atomic-result-template#singaporeTitle').conditions = [
-                  (result) => /singapore/i.test(result.title),
-                ];
-              </script>
-            </atomic-result-template>
-          </atomic-result-list>
+          <atomic-result-list fields-to-include="liboardinteractionstyle,ytthumbnailurl"> </atomic-result-list>
```

Comment:
can remove fields-to-include="liboardinteractionstyle,ytthumbnailurl" since they were used in the templates